spec/system/jobseekers/jobseekers_can_add_professional_status_to_their_profile_spec.rb

```diff
@@ -0,0 +1,139 @@
+        click_on "Save and continue"
+        within "ul.govuk-list.govuk-error-summary__list" do
+          expect(page).to have_link("Select yes if you have a teacher reference number. If you have qualified teacher status (QTS) you must select yes.", href: "#jobseekers-profile-qualified-teacher-status-form-has-teacher-reference-number-field-error")
+        end
+        within(find("fieldset", text: "Do you have a teacher reference number (TRN)?")) do
+          choose "No"
+        end
+        click_on "Save and continue"
+
+        expect_page_to_have_professional_status_information("no", nil, nil, nil)
+      end
+    end
+  end
+
+  describe "editing professional status" do
+    let!(:profile) { create(:jobseeker_profile, jobseeker:, qualified_teacher_status: "yes", qualified_teacher_status_year: "2020", teacher_reference_number: "7777777", statutory_induction_complete: "yes", has_teacher_reference_number: "yes") }
+
+    before { visit jobseekers_profile_path }
+
+    it "allows jobseekers to add professional status information" do
+      expect_page_to_have_professional_status_information("yes", "2020", "7777777", "yes")
+      click_on "Change qualified teacher status"
+
+      within(find("fieldset", text: "Do you have qualified teacher status (QTS)?")) do
+        expect(find("#jobseekers-profile-qualified-teacher-status-form-qualified-teacher-status-yes-field")).to be_checked
+      end
+      within(find("fieldset", text: "Do you have a teacher reference number (TRN)?")) do
+        expect(find("#jobseekers-profile-qualified-teacher-status-form-has-teacher-reference-number-yes-field")).to be_checked
+      end
+      expect(find("#jobseekers-profile-qualified-teacher-status-form-qualified-teacher-status-year-field").value).to eq("2020")
+      within(find("fieldset", text: "Have you completed your statutory induction period?")) do
+        expect(find("#jobseekers-profile-qualified-teacher-status-form-statutory-induction-complete-yes-field")).to be_checked
+      end
+      expect(find("#jobseekers-profile-qualified-teacher-status-form-teacher-reference-number-field").value).to eq("7777777")
+
+      fill_in "Year QTS was awarded", with: "2000"
+      fill_in "What is your Teacher reference number (TRN)?", with: "1234567"
+      choose "I'm on track to complete it"
+
+      click_on "Save and continue"
+
+      expect_page_to_have_professional_status_information("yes", "2000", "1234567", "on_track")
+
+      click_on "Change qualified teacher status"
+
+      within(find("fieldset", text: "Do you have qualified teacher status (QTS)?")) do
+        choose "No"
+      end
+
+      within(find("fieldset", text: "Do you have a teacher reference number (TRN)?")) do
+        choose "No"
+      end
+
+      click_on "Save and continue"
+
+      expect_page_to_have_professional_status_information("no", nil, "", nil)
+    end
+  end
+
+  def expect_page_to_have_professional_status_information(qts, year, trn, statutory_induction_complete)
```

Comment:
suggestion: 

Given the number of parameters, maybe we could convert them into named params for easier reading:
eg: `expect_page_to_have_professional_status_information(qts: "yes", year: "2000", trn: "1234567", statutory_induction_complete: "on_track")`
vs 
`expect_page_to_have_professional_status_information("yes", "2000", "1234567", "on_track")`